mesh/mesh.cpp

```diff
@@ -2774,6 +2774,41 @@ void Mesh::DoNodeReorder(DSTable *old_v_to_v, Table *old_elem_vert)
    Nodes->Update(); // just needed to update Nodes->sequence
 }
 
+void Mesh::SetPatchAttribute(int i, int attr)
+{
+   MFEM_ASSERT(NURBSext, "SetPatchAttribute is only for NURBS meshes");
+   NURBSext->SetPatchAttribute(i, attr);
```

Comment:
I do not directly understand see why we have to set the attribute to patchTopo and the elements separately (line 2780 and 2784 resp). Is this for different usage? And, shouldn't these in the future be connected in some kind or does it have to do with the way NURBS are currently set up in MFEM?